docs/features/vsan-policy.md

```diff
@@ -1,11 +1,55 @@
-# What is vSAN
-[vSAN](http://www.vmware.com/products/virtual-san.html) is a shared storage (all ESX nodes in a cluster can access the same vSAN store), hyper-converged (vSAN is integrated into vSphere and runs on the same hardware nodes that vSphere manages), flash optimized storage offering from VMware.
+###...
+
+# Introduction to vSAN
```

Comment:
Rename this section to "Storage Policy Based Management". 
It needs to explain the following -
1. Benefits of SPBM
2. Introduction to SPBM for VSAN
3. Detailed commands.

I will share the a writeup that explains #1 and #2 offline. #3 is already covered here.